### ISSUE TITLE
Homepage naming is inconsistent

### ISSUE THREAD

**Issue by andrewhick** [alphagov/wcag-primer#134]
The homepage is referred to as "Home" in the page title, "WCAG 2.1 Getting started" in heading 1 and "Overview" in the menu. This may cause confusion to some users especially when wanting to return to that page.

As it's all on the same page it probably doesn't fail WCAG AA (for example, Consistent Identification), but we should still fix it.
